content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -94,6 +95,21 @@ variables (map of keys)
     └── sensitive (boolean)
 ```
 
+### Value: `target_addrs`
+
+* **Value Type:** An array of strings representing [resource addresses](/docs/internals/resource-addressing.html).
+
+Provides the targets specified using the [`-target`](/docs/commands/plan.html#resource-targeting) flag in the CLI or the `target-addrs` attribute in the API. Will be undefined if no resource targets are specified.
+
+To prohibit targeted runs altogether, make sure the `target_addrs` value is undefined or empty:
+
+```
+import "tfrun"
+
+main = (length(tfrun.target_addrs) else 0) == 0
```

Comment:
the length() function still seems to be failing with undefined

FWIW, I think asserting `tfrun.target_addrs is null` should work, though maybe `tfrun.target_addrs is null or tfrun.target_addrs is []` would work? (I also dont know if thats proper sentinel syntax)

Reply:
I misread this! Carry on. ✅ ing